### ISSUE TITLE
Color functions not being processed

### ISSUE THREAD

**Issue by benfrain** [postcss/postcss-color-function#6]
Beginner question. I'm running PostCSS/postcss-color-function via Gulp and tried a reduction to get to the bottom of my issue. I have this task:

``` javascript
var postcss = require("gulp-postcss"); 
var colorFunction = require("postcss-color-function");

gulp.task("css", function(){
    var processors = [
    colorFunction
        ];
    gulp.src("preCSS/styles.css")
    .pipe(postcss(processors))
    .pipe(gulp.dest("output"));
});
```

Given this input:

``` css
.thing {
   color: color(white a(90%));
}
```

I am getting this (same) output:

``` css
.thing {
   color: color(white a(90%));
}
```

Is this due to the way I am configuring?

**Comment by MoOx** [comment]
this plugin doesn't use recent postcss api. Meanwhile, use

``` js
var processors = [
    colorFunction() // plugin is a function that generate a function
        ]
```

**Comment by MoOx** [comment]
Please keep this issue open. I will close when it will be "fixed".

**Comment by benfrain** [comment]
No problem. Thanks.

**Comment by MoOx** [comment]
Closed by 5682af25221b494fcded9463b38bdb439a155bae
